opus/import/obs_bundle_cassini_iss_fring_mosaics_rsfrench2025.py

```
@@ -0,0 +1,332 @@
+        fs = self._index_col('file_spec')
+        if fs is not None and 'data_reproj_img' in fs.lower():
+            return self._create_mult('IMG')
+        return self._create_mult('MOS')
+
+    ################################
+    ### OVERRIDE FROM ObsPdsPDS4 ###
+    ################################
+
+    def field_obs_pds_note(self):
+        raw = self._index_col('notes')
+        if raw is not None: raw = raw.strip()
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)